app/parse.py

```diff
@@ -15,8 +37,82 @@ class Product:
     num_of_reviews: int
 
 
-def get_all_products() -> None:
-    pass
+def parse_single_product(product_soup: BeautifulSoup) -> Product:
+    title_elem = product_soup.select_one(".title")
+    description_elem = product_soup.select_one(".description")
+    price_elem = product_soup.select_one(".price")
+    rating_elem = product_soup.select_one("p[data-rating]")
+    review_elem = product_soup.select_one(".review-count")
+
+    title = title_elem["title"] if title_elem else ""
+    description = description_elem.text if description_elem else ""
+    price = float(price_elem.text.replace("$", "")) if price_elem else 0.0
+    rating = int(rating_elem["data-rating"]) if rating_elem else 0
+    num_of_reviews = (
+        int(review_elem.text.split()[0].replace("$", "")) if review_elem else 0
+    )
+
+    return Product(
+        title=title,
+        description=description,
+        price=price,
+        rating=rating,
+        num_of_reviews=num_of_reviews,
+    )
+
+
+def get_all_products() -> [Product]:
+    driver = webdriver.Chrome()
+
+    for url_key, url_value in urls.items():
+        logging.info(f"Start process {url_key} page")
+        driver.get(url_value)
+
+        cookies = driver.find_elements(By.CLASS_NAME, "acceptCookies")
+
+        if cookies and cookies[0].is_displayed() and cookies[0].is_enabled():
+            cookies[0].click()
+
+        try:
+            button_more = driver.find_element(
+                By.CLASS_NAME, "ecomerce-items-scroll-more"
+            )
+            while button_more.is_enabled() and button_more.is_displayed():
+                button_more.click()
+                time.sleep(1)
```

Comment:
https://selenium-python.readthedocs.io/waits.html#explicit-waits
try to use WebDriverWait